Practica2/contacto.php

```diff
@@ -1,34 +1,14 @@
 <?php
-  session_start();
-   
-  $modoOscuro = isset($_COOKIE['modoOscuro']) && $_COOKIE['modoOscuro'] === 'activado';
-?>
-<!DOCTYPE html>
-<html lang="es">
-  <head>
-    <?php 
-        
-      if ($modoOscuro) {
-          echo '<link id ="estilo" rel="stylesheet" href="css/indexNight.css">';
-      }else{
-          echo '<link  id ="estilo"rel="stylesheet" href="css/index.css">';}
-    ?>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"/>
-    <title>Contacto</title>
-  </head>
-  
-  <body>
-    <div class="container-encabezado">
-    <?php include("componentes/header.php"); ?>
 
-    </div>
+require_once 'includes/config.php';
```

Comment:
Si quereis mantener este formulario de contacto, ahroa ya podéis hacerlo bien y gestionarlo como un formulario con PHP. Tenéis disponible la función mail de PHP que envía correos (podéis ver los que envieis en mail.containers.fdi.ucm.es)